docs/examples/make_pymvpa_collection.sh

```diff
@@ -44,37 +32,63 @@ datalad create-handle mnist
         # TODO --doi "10.1109/5.726791"
 )
 
-# Forrest gump dataset, although not yet used in PyMVPA stock materials,
-# analysis scripts use PyMVPA heavily so it is a worthwhile addition
-echo "I: adding forrest_gump dataset"
-datalad install-handle http://psydata.ovgu.de/forrest_gump/.git
-# Disabled for now - it's just a lot. Not needed for testing.
-#(
-#    cd forrest_gump
-#    echo "I: getting forrest_gump content"
-#    datalad get .
-#)
+# add the handle to the collection
+datalad add-handle mnist pymvpa
 
-# new plain collection
-echo "I: creating PyMVPA collection"
-datalad create-collection pymvpa_collection pymvpa # path name
+# Now, we want to add several already existing annexes/handles. There are
+# different possibilities to do this.
+# We want to publish the collection later on, including the handles. So we need
+# to make sure, that the to-be published handles grant access to their actual
+# content.
+#
+# First option:
+# Install that handle locally, get all of its content, that therefore can be
+# pushed on publishing. This approach is necessary only, if we actually want to
+# change the content or the origin is not publicly available or we want to
+# provide an additional copy of that content. This approach could be combined
+# with the second one, to link several sources of the content.
 
-# add all handles to the pymvpa_collection
-echo "I: adding all handles to the PyMVPA collection"
-for h in haxby2001 tutorial_data mnist forrest_gump; do
-    datalad add-handle "$h" pymvpa
-done
+echo "I: adding haxby2001"
+# Haxby 2001 -- the main demo dataset
+datalad install-handle http://data.pymvpa.org/datasets/haxby2001/.git
+(
+    cd haxby2001
+    echo "I: getting haxby2001 content"
+    datalad get .
```

Comment:
1. part of the reason why I would like to avoid re-publishing the content is that in the grant we promised not to redistribute the content for human data.  So if we come up with some (smaller) handle which we would use (e.g. we could initiate one for mnist dataset on data.pymvpa.org) and share content for that one